index.html

```diff
@@ -11,7 +11,7 @@
 <body>
     <header>
         <nav class="navbar">
-            <img src="images/logo.png" class="nav-branding" alt="Hadi's logo">
+            <img src="images/Logo.png" class="nav-branding" alt="Hadi's logo">
             <ul class="nav-menu">
                 <li class="nav-item"><a href="about.html" class="nav-link">About</a></li>
                 <li class="nav-item"><a href="#" class="nav-link">Clients</a></li>
```

Comment:
- [ ] [OPTIONAL]  The top short bar section with social media links, logout, and language links aligned to the right which is currently missing in your project. Kindly add it to your project as seen in the original design.